src/panda_eup/pbd_interface.py

```diff
@@ -8,15 +8,18 @@
 from panda_pbd.msg import UserSyncGoal, MoveToContactGoal, MoveToEEGoal
 from panda_pbd.srv import MoveFingersRequest, ApplyForceFingersRequest
 from sensor_msgs.msg import JointState
+import numpy as np
+import random
```

Comment:
You don't need random. Please use numpy as explained here https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.random.rand.html